cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -297,12 +297,12 @@
     ],
 )
 
-unmigrated_X = numpy.zeros([good_obs.shape[0], var_unmigrated.shape[0]], dtype=numpy.float32)
-adata_with_lables_unmigrated = anndata.AnnData(
+unmigrated_X = numpy.zeros([unmigrated_obs.shape[0], var_unmigrated.shape[0]], dtype=numpy.float32)
```

Comment:
I believe this was what you intended @Bento007 (not the typo fix from `umigrated_obs` => `unmigrated_obs`, but the reference to the unmigrated obs df rather than the "good" obs df). But everything was working by chance since each df had 2 rows.

Confirm?